commands/mr/update/mr_update.go

```diff
@@ -29,6 +29,7 @@ func NewCmdUpdate(f *cmdutils.Factory) *cobra.Command {
 			var err error
 			var actions []string
 			var ua *cmdutils.UserAssignments
+			var ur *cmdutils.UserAssignments
```

Comment:
I have used the same `UserAssignments` which is used for updating assignee, the only issuw with this is actions returned by the method is prefixed with `assigned %q`

Reply:
This has to be fixed to differentiate the `--assignee` flag from the `--reviewer` flag.

Reply:
The actions message is produced by the `UsersFromReplaces` function here:
https://github.com/profclems/glab/blob/c443358684b7ca29f43049a096acc05fd22b3e87/commands/cmdutils/cmdutils.go#L428-L430.
I will push a few commits to fix some of these changes